packages/cli/scripts/util/cdkHelpers.js

```diff
@@ -285,6 +214,119 @@ async function reTranspile() {
   return await getInputFilesFromEsbuildMetafile(metafile);
 }
 
+//////////////////////
+// Check CDK dep versions
+//////////////////////
+
+/**
+ * Check if the user's app is using the exact version of the currently supported
+ * AWS CDK version that Serverless Stack is using. If not, then show an error
+ * message with update instructions.
+ * More here
+ *  - For TS: https://github.com/aws/aws-cdk/issues/542
+ *  - For JS: https://github.com/aws/aws-cdk/issues/9578
+ */
+function runCdkVersionMatch(packageJson, cliInfo) {
+  const usingYarn = cliInfo.usingYarn;
+  const cdkVersion = cliInfo.cdkVersion;
+
+  // Check v1 dependencies
+  const v1Deps = [
+    ...getCdkV1Deps(packageJson.dependencies),
+    ...getCdkV1Deps(packageJson.devDependencies),
+  ];
+  if (v1Deps.length > 0) {
+    logger.error(
+      `\n${chalk.red("Update the following AWS CDK packages to v2:")}\n`
+    );
+    v1Deps.forEach((dep) => logger.error(chalk.red(`  - ${dep}`)));
+    logger.error(
+      `\nMore details on upgrading to CDK v2: https://github.com/serverless-stack/serverless-stack/releases/tag/v0.59.0\n`
+    );
+    throw new Error(`AWS CDK packages need to be updated.`);
+  }
+
+  const mismatchedDeps = getCdkV2MismatchedDeps(
+    packageJson.dependencies,
+    cdkVersion
+  );
+  const mismatchedDevDeps = getCdkV2MismatchedDeps(
+    packageJson.devDependencies,
+    cdkVersion
+  );
+
+  if (mismatchedDeps.length === 0 && mismatchedDevDeps.length === 0) {
+    return;
+  }
+
+  logger.info("");
+  logger.error(
+    `Mismatched versions of AWS CDK packages. Serverless Stack currently supports ${chalk.bold(
+      cdkVersion
+    )}. Fix using:\n`
+  );
+
+  if (mismatchedDeps.length > 0) {
+    const depString = formatDepsForInstall(mismatchedDeps, cdkVersion);
+    logger.info(
+      usingYarn
+        ? `  yarn add ${depString} --exact`
+        : `  npm install ${depString} --save-exact`
+    );
+  }
+  if (mismatchedDevDeps.length > 0) {
+    const devDepString = formatDepsForInstall(mismatchedDevDeps, cdkVersion);
+    logger.info(
+      usingYarn
+        ? `  yarn add ${devDepString} --dev --exact`
+        : `  npm install ${devDepString} --save-dev --save-exact`
+    );
+  }
+
+  logger.info(
+    `\nLearn more about it here — https://github.com/serverless-stack/serverless-stack#cdk-version-mismatch\n`
```

Comment:
Hey Frank. I encountered this today. This link doesn't go anywhere meaningful. I looked around but couldn't find a link that is more appropriate than `https://docs.serverless-stack.com/known-issues`. However, this doesn't seem like teh correct link either.

Reply:
Hey @aisflat439, nice catch! `https://docs.serverless-stack.com/known-issues` is the correct link.